### DISCUSSION TITLE
Unable to create Kafka Topic in confluent cloud.

### DISCUSSION THREAD

**Discussion by KPshanmuam** [trinodb/trino#13918] [Q&A]
Team,

              We are having issues to creating KAFKA topics in confluent cloud.  Getting below 401 error (unauthorized). Can you please advice how we can fix this issues.

We have already added topic keys into our GitLab vault

![image](https://user-images.githubusercontent.com/112157335/186842905-7963c067-785c-4daa-a400-d3daa9c40785.png)
.

**Comment by Praveen2112** [comment]
401 error (unauthorized) error might be due to missing or invalid username and password when we connect to SchemaRegistry. Trino doesn't support connecting with a secured SchemaRegistry service. Additionally Trino doesn't support creation of new topics to Kafka. Is the question is more focused on Terraform community ?

**Comment by KPshanmuam** [comment]
Hi Praveen, 

      Thanks for your update, we are not using Trino even i am not sure about that.
       i used terraform through the Gitlab pipeline. which i can able to create a environment in confluent cloud and getting error while creating topics.
       any sugissions would be greatly appriciated.
